src/libraries/Microsoft.Extensions.Logging.Console/ref/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -14,6 +14,10 @@
     <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\RequiresUnreferencedCodeAttribute.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'">
```

Comment:
I think this would be more version-resilient if we wrote it like this:

```xml
<ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))"
```

That way when `$(NetCoreAppCurrent)` == `net8.0` and `$(NetCoreAppMinimum)` == `net7.0`, we don't need to update this condition.

Reply:
Thanks